src/api/lib/activexml/transport.rb

```diff
@@ -397,7 +397,21 @@ def load_external_url(uri)
       content = nil
       proxyuri = ENV['http_proxy']
       proxyuri = CONFIG['http_proxy'] unless CONFIG['http_proxy'].blank?
```

Comment:
it would not be exactly the same, because it would also prefer a "" string in CONFIG over the environment, right?

Not sure if this is critical though ...